app/uk/gov/hmrc/cataloguefrontend/whatsrunningwhere/ReleasesConnector.scala

```diff
@@ -84,22 +83,38 @@ class ReleasesConnector @Inject()(
       }
   }
 
+  private val paginatedHistoryReads: HttpReads[PaginatedDeploymentHistory] = {
+    implicit val rf = JsonCodecs.deploymentHistoryReads
+    for {
+      history <- HttpReadsInstances.readFromJson[Seq[DeploymentHistory]]
+      resp    <- HttpReads.ask.map(_._3)
+      totals = resp.header("X-Total-Count").map(_.toLong).getOrElse(0L)
+    } yield PaginatedDeploymentHistory(history, totals)
+  }
+
   def deploymentHistory(
     environment: Environment,
-    from       : Option[Long]   = None,
-    to         : Option[Long]   = None,
-    team       : Option[String] = None,
-    app        : Option[String] = None
-  )(implicit
-    hc: HeaderCarrier
-  ): Future[Seq[Deployment]] = {
-    implicit val rf = JsonCodecs.deploymentReads
+    from: Option[Long]   = None,
+    to: Option[Long]     = None,
```

Comment:
can we replace Long with an iso-date everywhere? e.g. in the form/catalogue page urls too?